built_value_generator/test/built_value_generator_test.dart

```diff
@@ -872,6 +872,17 @@ abstract class _Value implements Built<_Value, _ValueBuilder> {
             isNot(contains(r'1.')),
           ));
     });
+
+    test(r'escape classes with $', () async {
```

Comment:
The unit tests are just for failures; for success tests I use end to end tests.

So, please remove this and instead add a value with a $ in the name in the end_to_end_test package, to values.dart and values_nnbd.dart, and run the generator.

Because the compile will fail if generation is broken, there is no need to actually add a test on top.

Thanks!

Reply:
should be ok now